src/Quiz.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+
+namespace Drupal\anu_lms;
+
+
+use Drupal\anu_lms_assessments\Entity\AssessmentQuestionResult;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\entity_reference_revisions\EntityReferenceRevisionsFieldItemList;
+use Drupal\node\NodeInterface;
+
+class Quiz extends Lesson
+{
+  const FIELD_NO_MULTIPLE_SUBMISSIONS = 'field_no_multiple_submissions';
+  const FIELD_QUESTION_RESPONSE = 'field_question_response';
+
+  /**
+   * Returns normalized data for Quiz page.
+   *
+   * @param EntityInterface $node
+   *   Lesson node.
+   *
+   * @return array
+   *   An array containing current node and referenced course.
+   */
+  public function getPageData(NodeInterface|EntityInterface $node)
+  {
+    $data = parent::getPageData($node);
+    return $this->_anu_lms_get_quiz_submission_data($node, $data);
+  }
+
+  /**
+   * Returns data for Quiz submissions.
+   *
+   * @param NodeInterface $node
+   *   Quiz node
+   *
+   * @param array $data
+   *   Data to be returned
+   */
+  protected function _anu_lms_get_quiz_submission_data(NodeInterface $node, array &$data)
```

Comment:
Naming for such methods should just be "getSubmissionData()" instead, according to the Drupal naming conventions.

Reply:
done